docs/supported_versions.md

```diff
@@ -0,0 +1,206 @@
+# OCP Version Support
+
+## MCE hub (management or hosting) Cluster
+[Multi-Cluster Engine](https://docs.openshift.com/container-platform/4.17/architecture/mce-overview-ocp.html) (MCE), which is available through OpenShift's OperatorHub, require specific OCP versions for the Management Cluster to remain in a supported state.
```

Comment:
> require specific OCP versions for the Management Cluster to remain in a supported state.

Is this just with respect to HyperShift or in general?

Reply:
This is MCE in general.